main.js

```diff
@@ -50,6 +50,23 @@ const createListView = (todos) => {
   });
 };
 
+// タスク表示/非表示切り替え機能の実装
+const radioSelect = () => {
+  if (radioAll.checked) {
+    todos.slice();
+    return createListView(todos);
```

Comment:
少し細かいのですが、この記述ですとcreateListViewの返却値をreturnするのが目的のように見えます。
このreturnは処理を終えるという意味で使用されると思うので、
```suggestion
    createListView(todos);
    return;
```
のような書き方が良いです！